debugging

```diff
@@ -0,0 +1 @@
+lib
```

Comment:
What is this about?
It looks like a non-intentional include in this commit, or do I miss something?

Reply:
this is just a symbloc link

Reply:
ah, did not recognize this, thanks!